dev/Repeater/SelectionNode.cpp

```diff
@@ -126,7 +126,8 @@ std::shared_ptr<SelectionNode> SelectionNode::GetAt(int index, bool realizeChild
             auto childData = m_dataSource.get().GetAt(index);
             if (childData != nullptr)
             {
-                auto resolvedChild = m_manager->ResolvePath(childData, weak_from_this());
+                auto const childDataIndexPath = winrt::get_self<class IndexPath>(IndexPath())->CloneWithChildIndex(index);
```

Comment:
>class [](start = 64, length = 5)

Is this class identifier needed? what does it do?

Reply:
This is needed to eliminate confusion between the type and the IndexPath function.